supporting-blog-content/evaluating-search-relevance-part-1/retrieve-and-rerank.ipynb

```diff
@@ -776,8 +777,29 @@
   {
    "metadata": {},
    "cell_type": "markdown",
-   "source": "Finally, in this section we show an implementation of confidence intervals for the `nDCG@10` metric using [bootstrapping](https://en.wikipedia.org/wiki/Bootstrapping_(statistics)) combined with the `percentile` method. It should be noted that this statistic is affected a lot by the number of queries in the dataset i.e. smaller confidence intervals are expected for larger query sets and vice versa",
-   "id": "e9fbbb8760ddc49d"
+   "source": "---",
+   "id": "1eae7142374e2f7a"
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "### Confidence intervals",
+   "id": "392ab084d5bc024"
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "In this section we will briefly touch upon the concepts of confidence intervals and statistical significance.\n",
+    "\n",
+    "We can think of it as follows: Our goal is to estimate the performance of our pipeline (retrieval and/or reranking) on a target corpus. Ideally, we would like to have access to **all** queries that our end-users will run against it but of course this is impossible. Instead, we have the set of test queries provided by the benchmark and we implicitly assume that the performance on this set can act as an accurate proxy of the overall performance (in the ideal scenario).\n",
+    "\n",
+    "But we can make some extra assumptions to increase the reliability of our analysis. [Confidence intervals](https://en.wikipedia.org/wiki/Confidence_interval) , a concept from statistical theory, give us a tool to handle our uncertainty: By setting a certain level of confidence, let's go with 95% in this example, we can derive a range of values that will likely contain the parameter of interest (here the performance in the **ideal** scenario). In other words, if we repeated the same process an infinite number of times (by drawing different test sets) we could be confident that that in 95% of them the confidence interval would encompass the true value.\n",
+    "\n",
+    "The code below shows an example of deriving confidence intervals using [bootstrapping](https://en.wikipedia.org/wiki/Bootstrapping_(statistics)) combined with the `percentile` method. It should be noted that this statistic is affected a lot by the number and the variability of queries in the dataset i.e. smaller confidence intervals are expected for larger query sets and vice versa\n",
```

Comment:
```suggestion
    "The code below shows an example of deriving confidence intervals using [bootstrapping](https://en.wikipedia.org/wiki/Bootstrapping_(statistics) combined with the `percentile` method. It should be noted that this statistic is affected a lot by the number and the variability of queries in the dataset i.e. smaller confidence intervals are expected for larger query sets and vice versa\n",
```

Reply:
Here, the right parenthesis `)` is part of Wikipedia's URL. I know it's a bit unfortunate :)

Reply:
ah, then we might have to escape it, since not it renders like this:
<img width="305" alt="image" src="https://github.com/user-attachments/assets/9e5fd5c6-46a3-4925-8185-3c742a501505">

Reply:
Right. I will do that 👍